edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/linkhandlers/ModuleMessageLinkHandler.cs

```diff
@@ -46,7 +46,7 @@ static class Metrics
             static readonly IMetricsCounter MessagesMeter = Util.Metrics.Metrics.Instance.CreateCounter(
                 "messages_sent",
                 "Messages sent to module",
-                new List<string> { "from", "to", "from_route_output", "to_route_input", "priority" });
+                new List<string> { "from", "to", "from_route_output", "to_route_input", "priority", "ms_telemetry" });
```

Comment:
I'm guessing you have considered an alternate approach where edge agent maintains the state to let it know what metrics it should scrape?

The downsides of this are that this list would need to be maintained, but then we wouldn't have to include this ms_telemetry data with every metric.